packages/skeleton-loader/src/Skeleton/Skeleton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+
+import { rootStyles, sizeStyles, themeStyles } from './Skeleton.styles';
+import { Size } from './Skeleton.types';
+import { SkeletonProps } from '.';
+
+export function Skeleton({
+  size = Size.Default,
+  darkMode,
+  className,
+  ...rest
+}: SkeletonProps) {
+  const { theme } = useDarkMode(darkMode);
+  return (
+    <div
+      className={cx(
+        rootStyles,
+        sizeStyles[size],
+        themeStyles[theme],
+        'lg-ui-skeleton',
```

Comment:
could you use `createUniqueClassName`?

Reply:
On second thought, I don't think we even need this. I'll remove it

I initially had it here because I thought every instance of Skeleton would need to share a className (so no createUniqueClassName), but we should be able to achieve the synchronized animation even without it